src/udev/udev-builtin-net_id.c

```diff
@@ -222,6 +222,7 @@ static int dev_pci_onboard(struct udev_device *dev, struct netnames *names) {
         const char *attr, *port_name;
         size_t l;
         char *s;
+        int ret = 0;
```

Comment:
all through our codebase we tend to call the variable we assign return code error numbers to `r`, and we I think we should follow this here, too. Also, no need to initialize this to 0…

Reply:
OK, fixed